nuxeo-path-suggestion/nuxeo-path-suggestion.html

```diff
@@ -212,10 +212,12 @@
       },
 
       _queryChildren: function(parent, term) {
-        this.query = 'SELECT * FROM Document WHERE ecm:parentId = \'' + parent.uid + '\' AND ecm:name LIKE ? ' +
-                     'AND ecm:mixinType = \'Folderish\' AND ecm:mixinType != \'HiddenInNavigation\' ' +
-                     'AND ecm:isVersion = 0 AND ecm:currentLifeCycleState != \'deleted\'';
-        this.params = [term+'%'];
+        this.params = {
+          queryParams: 'SELECT * FROM Document WHERE ecm:parentId = \'' + parent.uid + '\' ' +
+                       'AND ecm:name LIKE \'' + term + '%\' AND ecm:mixinType = \'Folderish\' ' +
```

Comment:
Unrelated to your commit but the `term` should be escaped before being added to the NXQL string.